README.md

```diff
@@ -2,53 +2,92 @@
-```
+Acceptance Workflows:
+
+1. Feedback: provide feedback on validation and review (successful / unsuccessful validation).
+2. Update DB: update simulation model database with new parameter (successful validation).
+3. Document: document all derivation and validation steps.
+
+The implementation in *simtools-workflows* consist of the following main components:
+
+- **workflows** encoded in [Common Workflow Language (CWL)](https://www.commonwl.org/) (see [./workflows](./workflows) in this repository)
+- **parameter schemas** defining all input data and simulation model parameters (see [./schemas](./schemas) in this repository)
+- **simulation tools and software** consisting of [simtools](https://github.com/gammasim/simtools) and the simulation software (e.g., [CORSIKA](https://www.iap.kit.edu/corsika/) and [sim_telarray](https://www.mpi-hd.mpg.de/hfm/~bernlohr/sim_telarray/)).
+
+## Workflows and tools
+
+CWL workflows consists of
+
+- *Tools* are called by *steps* in a *Workflow* and are doing one single task. In most cases, tools are calling an application of [simtools](https://github.com/gammasim/simtools) including all required configuration parameters (e.g., [workflows/tools/derive_array_elements_coordinates.cwl](workflows/tools/derive_array_elements_coordinates.cwl)).
+- *Workflows* connect *tools*, and allow to execute the steps discussed above (receive, assert, derive, validate, ...), see.e.g, workflow/DeriveArrayElementCoordinates.cwl](./workflows/DeriveArrayElementCoordinates.cwl).
+
+### Getting started
+
+Clone the [simtools workflows](https://github.com/gammasim/workflows) repository and install dependencies using mamba or conda:
+
+```bash
+git clone git@github.com:gammasim/workflows.git
 mamba env create -f environment.yml
-conda activate simtools-workflows-dev
+mamba activate simtools-workflows-dev
 ```
 
-## Running examples
+All examples require docker to be installed and running.
+Tools are using the [simtools-prod:latest](https://github.com/gammasim/simtools/pkgs/container/simtools-prod) docker image.
+
+### Example workflow
 
 The following command line tool converts telescope coordinates from UTM to CORSIKA coordinates.
-```
+
+```bash
 cwltool DeriveArrayElementCoordinates.cwl ../tests/resources/test_derive_array_elements_coordinates.yml
 ```
-The workflow steps are:
+
+The workflow steps executed are:
+
 - assert input data
 - convert coordinates
 - assert derive parameter values
 
-(see the log files produced)
+Expected output:
 
-## CWL validation
+- telescope position file in CORSIKA coordinates (ecsv format)
+- metadata files describing each workflow step (yml format)
+- log files (stdout and stderr; ascii format)
 
-Use `cwltool --validate file_name.cwl` to check a workflow or command line tool for valid CWL.
+### CWL validation
 
-## Graphs
+Use `cwltool --validate file_name.cwl` to check a workflow file or command line tool to valid CWL code (this is also done by the CI).
+
+### Workflow Graphs
```

Comment:
As discussed, this is part of the code organisation / validation, not of the workflow validation.

Reply:
Ok, maybe you show me one of the graphs next week again so that I get a better understanding of what they are. I tried to open the link with the example you provided in the text and got an error: `Error: The workflow could not be retrieved from the Git repository using the details given`

Reply:
Fixed the path to the graph.